debian/rules

```diff
@@ -29,6 +29,7 @@ override_dh_builddeb:
 override_dh_auto_install:
 	# Our own tests
 	debian/test_build_sanity.sh
+	# python3 setup.py install --force --root=debian/kolibri --no-compile -O0 --prefix=/usr --install-lib=/usr/lib/python3/dist-packages
```

Comment:
Maybe we delete this line rather than comment out?

Reply:
Done, I was keeping it for reference of an example when dh_python can not be used, but it can be easily recovered from git if needed.